rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -418,6 +418,19 @@ public boolean removeJob(String sessionId, String jobId) throws RestException {
         }
     }
 
+    @Override
+    public boolean removeJobs(String sessionId, List<String> jobsId) throws RestException {
+        try {
+            // checking permissions
+            checkAccess(sessionId, "DELETE jobs/");
+
+            Scheduler s = checkAccess(sessionId, "DELETE jobs/" + jobsId.get(0));
```

Comment:
You should check if jobsId is empty or not.

Reply:
oh. you are right. thanks